lib/migrator/migrationSteps.spec.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-env jest */
+
+const path = require('path')
+const fse = require('fs-extra')
+const reporter = require('./reporter')
+const fileHelpers = require('./fileHelpers')
+const config = require('./../config')
+const myReporter = require('./reporter')
```

Comment:
We're requiring reporter twice here, is that on purpose?

Reply:
I've removed the myReporter require and replaced the uses with reporter.